docs/_docs/SQL/ODBC/odbc-driver.adoc

```diff
@@ -201,21 +201,38 @@ As a result, `ignite-odbc-amd64.msi` and `ignite-odbc-x86.msi` files should appe
 
 On a Linux-based operating system, you will need to install an ODBC Driver Manager of your choice to be able to build and use the Ignite ODBC Driver. The ODBC Driver has been tested with link:http://www.unixodbc.org[UnixODBC].
 
-Additionally, you will need `GCC`, `G++`, and `make` to build the driver and its dependencies.
+==== Prerequisites
+include::includes/cpp-linux-build-prerequisites.adoc[]
 
-Once all the above mentioned are installed, you can build the Ignite ODBC driver:
+NOTE: JDK is needed currently only for building, odbc driver binary doesn't depend on it.
 
-[source,shell]
+==== Building ODBC driver
+- Create build directory for cmake. We'll refer to this as `{CPP_BUILD_DIR}`
```

Comment:
I would update the steps this way if you don't mind:

-  Create a build directory for CMake. We'll refer to it as to `{CPP_BUILD_DIR}`.
 - (Optional) Select the prefix of the installation directory which defaults to `/usr/local` if nothing is set. We'll refer to this directory as to `{CPP_INSTALL_DIR}`.
 - Build and install the driver by executing the following commands:

Reply:
Fixed